nginx/config.go

```diff
@@ -205,6 +205,11 @@ http {
 			allow 127.0.0.1;
 			deny all;
 		}
+    location /nginx_status {
+      stub_status on;
+      allow 127.0.0.1;
+      deny all;
+    }
```

Comment:
We all use different editors. I use Sublime Text 3, I think @krancour uses VSCode, others use Atom. I think those three editors covers a large percentage of the Deis team.